<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CJImport</class>
 <widget class="QDialog" name="CJImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成绩录入</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>73</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>学号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="searchEdit">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>输入学号、姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>125</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>121</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>66</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="classEdit">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>173</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>58</x>
     <y>177</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>班级:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>255</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>德育成绩:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DYEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>251</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>305</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>智育成绩:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ZYEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>301</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="failCheck">
   <property name="geometry">
    <rect>
     <x>-33</x>
     <y>351</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>不及格科目: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>370</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>提交修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>370</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>214</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>学期:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>上学期</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>下学期</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
